eng/pipelines/runtime.yml

```diff
@@ -691,6 +691,7 @@ jobs:
     runtimeFlavor: mono
     platforms:
     - OSX_x64
+    - Linux_arm64
```

Comment:
Can we open an issue to track removing these steps and using the coreclr build-test steps instead? Of course the best will be to wait for: https://github.com/dotnet/runtime/pull/35783 to land.

Reply:
Ok, here it is: https://github.com/dotnet/runtime/issues/35979

Reply:
Thanks.